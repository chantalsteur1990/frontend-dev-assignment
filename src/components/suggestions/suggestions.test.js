import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Suggestions from './suggestions';

configure({ adapter: new Adapter() });

const mockProps = {
  suggestions: [{
    searchterm: 'mooie trui',
    nrResults: 1203
  }],
  query: 'tr',
  handleSelect: jest.fn()
};

describe('Suggestions', () => {
  it('should render (snapshot)', () => {
    const component = shallow(<Suggestions {...mockProps} />);
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
