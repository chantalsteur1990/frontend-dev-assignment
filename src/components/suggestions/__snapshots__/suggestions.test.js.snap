// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Suggestions should render (snapshot) 1`] = `
<div
  className="suggestions"
>
  <div
    className="suggestions__suggestion"
    dangerouslySetInnerHTML={
      Object {
        "__html": "<span class=\\"bold\\">mooie </span>tr<span class=\\"bold\\">ui</span> (1203)",
      }
    }
    id="mooie trui"
    key="mooie trui"
    onClick={[Function]}
  />
</div>
`;
